framework/test/jest/.eslintrc.json

```diff
@@ -1,26 +1,9 @@
 {
-	"extends": ["plugin:jest/recommended"],
-	"plugins": ["jest"],
-	"env": {
-		"jest/globals": true
-	},
+	"extends": ["lisk-base/jest"],
 	"rules": {
 		"callback-return": "error",
 		"no-unused-expressions": "off",
 		"import/no-extraneous-dependencies": ["error", { "devDependencies": true }],
```

Comment:
this rule is also in bas/jest config.

https://github.com/LiskHQ/eslint-config-lisk-base/blob/master/jest.js#L7